torch/csrc/jit/frontend/tracer.cpp

```diff
@@ -1,6 +1,7 @@
 #include <torch/csrc/jit/frontend/tracer.h>
 
 #include <ATen/Backtrace.h>
+#include <ATen/ScalarOps.h>
```

Comment:
Why was this needed?

Reply:
previously, we rely on `#include <torch/csrc/autograd/function.h>` which `#include <torch/csrc/autograd/input_buffer.h>`, which `#include <ATen/ATen.h>`.

This header is needed for 
https://github.com/pytorch/pytorch/blob/fa3aec70ca55c237529c63912a27a93f4ebc7f5d/torch/csrc/jit/frontend/tracer.cpp#L910